data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1179 @@
+
+
+			label cases
+			`	"Imagine, if you will, during the substantial conflicts the humans call the Turf Wars, a hostile alien came to human space. We Quarg woefully witnessed that hazardous conflict - it was a time of great waste, unfortunate peril, and unsavory distrust among humanity. The Gegno are in a vulnerable state notably comparable to such a time, and introduction of even more hostilities would not befit the case, especially of those from the outside. We have been contacted by numerous members of high Gegno standing and wish for it to be matter only for further strict cautionaries, but regrettably, the Gegno believe it a matter beyond intricate attentiveness." The Quarg leaves its chair and motions you to do the same.`
+			choice
+				`	(Stand up.)`
+
+			`	"You are recommended for departure from Gegno space as of this decisive moment. Meanwhile, we shall attempt a most careful negotiation with their hierarchy to come in terms of an acceptable toleration of storts. Should we find commonality, you will receive communication from one of the title 'Adrauni', as has been bestowed upon me."`
+			`	As the rest of the Quarg escort you out of the room, the Quarg named 'Adrauni' addresses you one last time. "Do avoid any further, violent tendencies, captain." If the Gegno are willing to go as far as to contact the Quarg, then this is truly an important matter to them. For the time being, it's probably best to listen to the Quarg's warnings this time around and leave Gegno space until some kind of resolution is found, if at all.`
+				decline
+
+	on decline
+		event "gegno: quarg talks" 11
+
+
+mission "Under the Nose"
+#	invisible
+	landing
+	source "Giaru Gegno"
+	to offer
+		has "Gegno Suspicions: declined"
+		not "Gegno Genocide Defense: offered"
+		not "yvx killed"
+	to complete
+		never
+	on enter "Aleit"
+		fail
+
+	npc
+		on kill
+			set "yvx killed"
+		government "Gegno Vi"
+		personality heroic vindictive ramming daring entering fleeing uninterested
+		system "Aleit"
+		ship "Rhyolite" "Imtydei"
+
+
+mission "Scin Surveillance"
+#	invisible
+	repeat
+	landing
+	deadline 7
+	source
+		near "Heutesl" 100
+	to offer
+		random < 70
+		has "event: gegno: suspicions"
+		not "Scin Suspicion: offered"
+		not "Gegno Genocide Defense: offered"
+		not or
+			"Gegno Anticipation: offered"
+			"Gegno Intervention: offered"
+	to complete
+		never
+	npc save
+		government "Gegno Scin"
+		personality surveillance
+		system
+			government "Gegno Scin"
+		fleet
+			names "gegno scin"
+			variant
+				"Tridymite"
+
+
+mission "Veering Vi A"
+	repeat
+#	invisible
+	landing
+	deadline 7
+	to offer
+#		random < 30
+		has "Gegno Suspicions: declined"
+		not "Full Vi Backlash: offered"
+		not "Gegno Genocide Defense: offered"
+		not or
```

Comment:
that works?

Reply:
As far as I've tested, but I'll triple check.

Reply:
maybe it'd be clearer if it was indented, or like just the logic could be inverted to use and, given that

not or
    y
    x
is the same as
not y
not x

Because with ES by default it's ands and you're checking for both of them (and) not being smth

Reply:
Having it indented and lower fails the data check.